src/comgr.cpp

```diff
@@ -991,6 +991,19 @@ void BuildHip(const std::string& name,
            }))
             opts.push_back("-std=c++17");
 
+        const char* rocm_path = std::getenv("ROCM_PATH");
```

Comment:
There is an error on this line for the checker, I think we need to use the MIOpen Macros instead.

Ex:
```c++
MIOPEN_DECLARE_ENV_VAR_STR(MIOPEN_CUSTOM_CACHE_DIR)
...
const auto& custom = env::value(MIOPEN_CUSTOM_CACHE_DIR);
```